src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.Sdk.targets

```diff
@@ -140,6 +140,11 @@ Copyright (c) .NET Foundation. All rights reserved.
           Inputs="$(ProjectAssetsFile);$(UserRuntimeConfig)"
           Outputs="$(ProjectRuntimeConfigFilePath);$(ProjectRuntimeConfigDevFilePath)">
 
+    <PropertyGroup>
+      <DeveloperStoreLocation>$(HOME)</DeveloperStoreLocation>
```

Comment:
Does it make sense to reuse `DefaultComposeDir` here?  Instead of duplicating this location between here and ComposeStore.targets.